src/styles/blocks/card-list.scss

```diff
@@ -0,0 +1,25 @@
+.main {
+  &__card-list {
+    display: flex;
+    justify-content: center;
```

Comment:
![image](https://user-images.githubusercontent.com/102512764/201284657-d4ae6bfd-1b1d-43d0-b592-9683aa40a166.png)

Cards are not supposed to behave like this if you remove one of them. Try to remove the justify content and fix the height of the cards so that their height does not break when there are not 8 of them.